--- old/ompi/etc/openmpi-mca-params.conf	2023-06-22 16:14:39.701714371 +0900
+++ new/ompi/etc/openmpi-mca-params.conf	2023-06-22 16:39:40.581697838 +0900
@@ -59,8 +59,9 @@
 # parameters available and their default values.
 #rmaps_base_mapping_policy = dist:auto
 coll = ^ml
-hwloc_base_binding_policy = core
-btl = self
+#hwloc_base_binding_policy = core
+hwloc_base_binding_policy = none
+#btl = self
 pml_ucx_tls = any
 pml_ucx_devices = any
 opal_common_ucx_opal_mem_hooks = 0
@@ -93,3 +94,4 @@
 coll_tuned_scatter_min_procs               = 1048510
 coll_tuned_scatter_algorithm_max_requests  = 64
 
+btl_openib_allow_ib = 1
